### PR TITLE
Update MARKETPLACE_TOOLS_TAG to latest release

### PR DESCRIPTION
**Pull request by eshiroma**
/gcbrun

### CODE REVIEW

**Comment by eshiroma** [comment]
(Also add missing default value for a previously added required property)

File: MARKETPLACE_TOOLS_TAG [hunk 1]
```diff
@@ -1 +1 @@
-0.8.0
+0.9.2
```
Comment (vcanaa) [inline MARKETPLACE_TOOLS_TAG]: lineend

Comment (eshiroma) [reply MARKETPLACE_TOOLS_TAG]: Done.

Comment (eshiroma) [reply MARKETPLACE_TOOLS_TAG]: Replaced - it turns out that the newline causes this tag not to be recognized by CloudBuild. You can replace later if you figure out how to make it work :)

**Pull request merged**